src/_includes/cloud/enable-ssh.md

```diff
@@ -15,89 +15,31 @@ For more information on SSH keys, see the following:
 
 ## Locate an existing SSH key pair {#existing}
 
-An existing SSH key pair is typically located in the `.ssh` subdirectory of the user's home directory. This folder is hidden and may not display in the file manager or finder unless configured to display hidden files and folders.
-
-You can quickly verify if you have SSH keys by entering commands using terminal access.
-
-To check for SSH keys, enter the following command:
-
-```bash
-ls ~/.ssh
-```
-
-If you have SSH keys, a directory listing is displayed similar to the following:
-
-```terminal
-id_rsa  id_rsa.pub  known_hosts
-```
-
-If you do not have SSH keys, you need to generate the keys for adding to your Magento ECE account and GitHub account. See [Create a new SSH key pair](#ssh-create-new-key-pair).
-
-If you already have SSH keys, continue to:
-
--  [Add a public SSH key to your Magento account](#ssh-add-to-account) section
--  [Add your SSH key to your GitHub account](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/)
-
-## Create a new SSH key pair {#ssh-create-new-key-pair}
-
-Use the `ssh-keygen` command to create an SSH key pair. `ssh-keygen` is typically installed on Linux systems.
+An existing SSH key pair is typically located in the `.ssh` subdirectory of the user home directory. This folder is hidden and may not display in the File Manager or Finder if your system is not configured to display hidden files and folders.
 
 {:.procedure}
-To create an SSH key pair:
+Check for SSH keys:
```

Comment:
> At first, I created a suggestion to change this to "To check for existing SSH key:" and realized this is all straight from Github. I'm glad to see most of this removed, since we don't need to walk through Github instructions. In fact, you could probably just lead with line 37 and provide links to GitHub for the checks and the generate instructions.
> 
> But, if you keep it in, then for consistency of all procedure titles, I do suggest the following:

I thought about leaving the test procedure out, but it's not included directly on the GitHub pages for adding a key.  And if the key is already there, you don't have to go to GitHub.   I didn't think it hurt to leave it.